frontend/src/static/js/v_summary.js

```diff
@@ -543,6 +543,14 @@ window.vSummary = {
     getGroupCommitsVariance(total, group) {
       return total + group[this.sortingOption];
     },
+
+    getGroupTotalContribution(group) {
+      let totalContribution = 0;
+      group.forEach((ele) => {
+        totalContribution += ele.totalCommits;
```

Comment:
can use `.reduce` to make it a 1 line solution.

Reply:
done